hangouts_chat.py

```diff
@@ -320,6 +335,12 @@ def handle_event_MESSAGE(self, data):
             'space_id': data['space']['name'],
             'thread_id': data['message']['thread']['name']
         }
+
+        if 'attachment' in data['message']:
+            context['attachment'] = data['message']['attachment']
+        # pass httplib2.Http() authenticated handler to errbot. uselful to download attachments
```

Comment:
Maybe worth adding an example to the README of how to use this? I totally believe this statement is true, it's just hard to intuit why this code is sticking this object in the context (to me at least).

Reply:
Makes sense. Added an example to the README.